### ISSUE TITLE
GCPy question

### ISSUE THREAD

**Issue by TylerElgiar** [geoschem/gcpy#189]
## Type your question below:

I am new to using GCPy. After following the instructions listed on the "gcpy.readthedocs.io" page, I tried following instructions to make a plot. When I type in the command "import gcpy.plot as gcplot" I receive the following error: 

ModuleNotFoundError: No module named 'ESMF'

I'm not quite sure how to solve this issue. Is it as simple as installing xESMF from this website "xesmf.readthedocs.io"? I just want to double check before I go installing anything else.

**Comment by yantosca** [comment]
Thanks for reaching out @TylerElgiar.  How were you able to solve the problem?

**Comment by TylerElgiar** [comment]
I tried uninstalling GCPy and all of the dependencies, and tried reinstalling GCPy the "for developers" way. It still gave me the same error message, except this time the error was accompanied by a specific version of ESMF. So I followed instructions on the xesmf read the docs page to manually install xesmf using "conda install -c conda-forge xesmf=8.1.1". It gave me another error message that I can't remember exactly, but it started working for me after those steps.

-Tyler     On Wednesday, January 4, 2023, 11:46:07 AM MST, Bob Yantosca ***@***.***> wrote:  
 
 


Thanks for reaching out @TylerElgiar. How were you able to solve the problem?
